games-roguelike/adom/adom-1.1.1-r1.ebuild

```diff
@@ -11,26 +10,21 @@ SRC_URI="https://www.adom.de/adom/download/linux/${P//.}-elf.tar.gz"
 LICENSE="adom"
 SLOT="0"
 KEYWORDS="~amd64 ~x86"
-IUSE=""
 RESTRICT="strip" #bug #137340
-QA_FLAGS_IGNORED="${GAMES_PREFIX_OPT:1}/bin/adom"
 
 RDEPEND="sys-libs/ncurses-compat:5[abi_x86_32(-)]"
 
 S=${WORKDIR}/${PN}
 
 src_install() {
-	exeinto "${GAMES_PREFIX_OPT}/bin"
+	exeinto "/opt/bin"
 	doexe adom
 
-	keepdir "${GAMES_STATEDIR}/${PN}"
-	echo "${GAMES_STATEDIR}/${PN}" > adom_ds.cfg
+	keepdir "/var/games/${PN}"
+	echo "/var/games/${PN}" > adom_ds.cfg
 	insinto /etc
 	doins adom_ds.cfg
 
 	edos2unix adomfaq.txt
```

Comment:
`edos2unix` requires the `edos2unix` eclass. (was before implicit inherit by `eutils`)